Documentation/contributor-guide/release.md

```diff
@@ -5,15 +5,15 @@ The guide talks about how to release a new version of etcd.
 The procedure includes some manual steps for sanity checking, but it can probably be further scripted. Please keep this document up-to-date if making changes to the release process.
 
 ## Release management
```

Comment:
Nit - Markdownlint says there should be blank lines under headers.

Reply:
Done.